mozilla-release/browser/themes/shared/incontentprefs/preferences.inc.css

```diff
@@ -56,6 +56,10 @@ label {
   margin-bottom: 0 !important;
 }
 
+description > b {
```

Comment:
Need this to explicitly make `Forget window` bold

![image](https://user-images.githubusercontent.com/5318732/45024159-ad7b5b80-b038-11e8-8633-00814c6bb5ad.png)